eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '**/*.bicep'
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  - stage: LintBicepFiles
+    displayName: Lint Bicep Files
+
+    pool:
+      vmImage: azsdk-pool-mms-ubuntu-2204-1espt
+
+    jobs:
+      - job: LintBicep
```

Comment:
Do we need to fix all warnings? do we want to have a baseline and warn about any new issues?

Reply:
We ideally should fix all warnings. I am saving the actual data but we can always filter it out when we process it.